<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>868</width>
    <height>568</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>JM60 Electric Control System</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/graphics/connection.svg</normaloff>:/graphics/connection.svg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="opaqueResize">
       <bool>true</bool>
      </property>
      <property name="handleWidth">
       <number>2</number>
      </property>
      <widget class="QTabWidget" name="tabWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>2</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>350</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Appliances</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>4</number>
         </property>
         <property name="margin">
          <number>2</number>
         </property>
         <item>
          <widget class="ecsTableView" name="appliancesView">
           <property name="contextMenuPolicy">
            <enum>Qt::CustomContextMenu</enum>
           </property>
           <property name="dragEnabled">
            <bool>true</bool>
           </property>
           <property name="dragDropMode">
            <enum>QAbstractItemView::DragOnly</enum>
           </property>
           <property name="verticalScrollMode">
            <enum>QAbstractItemView::ScrollPerPixel</enum>
           </property>
           <property name="horizontalScrollMode">
            <enum>QAbstractItemView::ScrollPerPixel</enum>
           </property>
           <property name="sortingEnabled">
            <bool>true</bool>
           </property>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Control Groups</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="margin">
          <number>2</number>
         </property>
         <item>
          <widget class="ecsTableView" name="cGroupView">
           <property name="contextMenuPolicy">
            <enum>Qt::CustomContextMenu</enum>
           </property>
           <property name="dragEnabled">
            <bool>true</bool>
           </property>
           <property name="dragDropMode">
            <enum>QAbstractItemView::DragOnly</enum>
           </property>
           <property name="horizontalScrollMode">
            <enum>QAbstractItemView::ScrollPerPixel</enum>
           </property>
           <property name="sortingEnabled">
            <bool>true</bool>
           </property>
           <property name="changeAffectsGraphics" stdset="0">
            <bool>true</bool>
           </property>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="ecsGraphicsView" name="graphicsView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>6</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>200</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="renderHints">
        <set>QPainter::Antialiasing|QPainter::HighQualityAntialiasing|QPainter::TextAntialiasing</set>
       </property>
       <property name="dragMode">
        <enum>QGraphicsView::ScrollHandDrag</enum>
       </property>
       <property name="viewportUpdateMode">
        <enum>QGraphicsView::FullViewportUpdate</enum>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>868</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEvents">
    <property name="title">
     <string>Events</string>
    </property>
    <addaction name="actionSingle_Click"/>
    <addaction name="actionDouble_Click"/>
    <addaction name="actionPress_Hold"/>
    <addaction name="actionRelease"/>
    <addaction name="actionSignal_Change"/>
   </widget>
   <widget class="QMenu" name="menuActions">
    <property name="title">
     <string>Actions</string>
    </property>
    <addaction name="actionToggle_On_Off"/>
    <addaction name="actionSwitch_Color"/>
    <addaction name="actionStart_Fade"/>
    <addaction name="actionStop_Fade"/>
    <addaction name="actionSwitch_On"/>
    <addaction name="actionSwitch_Off"/>
   </widget>
   <widget class="QMenu" name="menuNetwork">
    <property name="title">
     <string>Network</string>
    </property>
    <addaction name="actionOpen_Connection"/>
    <addaction name="actionShow_Monitor"/>
    <addaction name="actionClose_Connection"/>
    <addaction name="actionUpload_to_Yacht"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuNetwork"/>
   <addaction name="menuEvents"/>
   <addaction name="menuActions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar_Main">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>34</width>
     <height>34</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionSave_As"/>
   <addaction name="separator"/>
   <addaction name="actionUpload_to_Yacht"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QToolBar" name="toolBar_Events">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSingle_Click"/>
   <addaction name="actionDouble_Click"/>
   <addaction name="actionPress_Hold"/>
   <addaction name="actionRelease"/>
   <addaction name="actionSignal_Change"/>
  </widget>
  <widget class="QToolBar" name="toolBar_Actions">
   <property name="windowTitle">
    <string>toolBar_3</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionToggle_On_Off"/>
   <addaction name="actionSwitch_Color"/>
   <addaction name="actionStart_Fade"/>
   <addaction name="actionStop_Fade"/>
   <addaction name="actionSwitch_On"/>
   <addaction name="actionSwitch_Off"/>
   <addaction name="actionRun_Actuator"/>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>About...</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset>
     <normalon>:/graphics/open.svg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset>
     <normalon>:/graphics/save.svg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset>
     <normalon>:/graphics/file-save-as.svg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset>
     <normalon>:/graphics/levels.svg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Settings...</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt...</string>
   </property>
  </action>
  <action name="actionSingle_Click">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/graphics/click-single.svg</normaloff>:/graphics/click-single.svg</iconset>
   </property>
   <property name="text">
    <string>Single Click</string>
   </property>
  </action>
  <action name="actionDouble_Click">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/graphics/click-double.svg</normaloff>:/graphics/click-double.svg</iconset>
   </property>
   <property name="text">
    <string>Double Click</string>
   </property>
  </action>
  <action name="actionPress_Hold">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/graphics/click-hold.svg</normaloff>:/graphics/click-hold.svg</iconset>
   </property>
   <property name="text">
    <string>Press &amp; Hold</string>
   </property>
  </action>
  <action name="actionRelease">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/graphics/click-release.svg</normaloff>:/graphics/click-release.svg</iconset>
   </property>
   <property name="text">
    <string>Release</string>
   </property>
  </action>
  <action name="actionToggle_On_Off">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/graphics/onoff.svg</normaloff>:/graphics/onoff.svg</iconset>
   </property>
   <property name="text">
    <string>Toggle On/Off</string>
   </property>
  </action>
  <action name="actionSwitch_On">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/graphics/bulb-lit.svg</normaloff>:/graphics/bulb-lit.svg</iconset>
   </property>
   <property name="text">
    <string>Switch On</string>
   </property>
  </action>
  <action name="actionSwitch_Off">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/graphics/bulb.svg</normaloff>:/graphics/bulb.svg</iconset>
   </property>
   <property name="text">
    <string>Switch Off</string>
   </property>
  </action>
  <action name="actionStart_Fade">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/graphics/fade-start.svg</normaloff>:/graphics/fade-start.svg</iconset>
   </property>
   <property name="text">
    <string>Start Fade</string>
   </property>
  </action>
  <action name="actionStop_Fade">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/graphics/fade-stop.svg</normaloff>:/graphics/fade-stop.svg</iconset>
   </property>
   <property name="text">
    <string>Stop Fade</string>
   </property>
  </action>
  <action name="actionSwitch_Color">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/graphics/connections.svg</normaloff>:/graphics/connections.svg</iconset>
   </property>
   <property name="text">
    <string>Switch Color</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionSignal_Change">
   <property name="icon">
    <iconset>
     <normalon>:/graphics/signal.svg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Signal Change</string>
   </property>
  </action>
  <action name="actionRun_Actuator">
   <property name="icon">
    <iconset>
     <normalon>:/graphics/actuator.svg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Run Actuator</string>
   </property>
  </action>
  <action name="actionOpen_Connection">
   <property name="icon">
    <iconset>
     <normalon>:/graphics/connect.svg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Open Connection</string>
   </property>
  </action>
  <action name="actionClose_Connection">
   <property name="icon">
    <iconset>
     <normalon>:/graphics/disconnect.svg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Close Connection</string>
   </property>
  </action>
  <action name="actionShow_Monitor">
   <property name="icon">
    <iconset>
     <normalon>:/graphics/eye.svg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Show Monitor</string>
   </property>
  </action>
  <action name="actionUpload_to_Yacht">
   <property name="icon">
    <iconset>
     <normalon>:/graphics/yachtnet.svg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Upload to Yacht</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ecsGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>ecsGraphicsView.h</header>
  </customwidget>
  <customwidget>
   <class>ecsTableView</class>
   <extends>QTableView</extends>
   <header>ecsTableView.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
